src/pocketmine/network/mcpe/protocol/AvailableCommandsPacket.php

```diff
@@ -57,19 +57,20 @@ class AvailableCommandsPacket extends DataPacket implements ClientboundPacket{
 	public const ARG_TYPE_FLOAT           = 0x02;
 	public const ARG_TYPE_VALUE           = 0x03;
 	public const ARG_TYPE_WILDCARD_INT    = 0x04;
-	public const ARG_TYPE_TARGET          = 0x05;
-	public const ARG_TYPE_WILDCARD_TARGET = 0x06;
+	public const ARG_TYPE_OPERATOR        = 0x05;
+	public const ARG_TYPE_TARGET          = 0x06;
 
-	public const ARG_TYPE_STRING   = 0x0f;
```

Comment:
does string just not exist anymore then...?

Reply:
Sorry, forgot to add it. I've added it in the last commit.